### ISSUE TITLE
Feature Request: ChangeCallBack feature

### ISSUE THREAD

**Issue by System64** [stevermeister/ngWig#134]
Hi,
Wouldn't it be great to have a changeCallBack option, alike pasteCallBack. 
I have modified the code and have implemented change callback by binding "DOMSubtreeModified" event with debounce, so that it won't get called so often. Is it a good approach, any suggestion would be good.

**Comment by bampakoa** [comment]
Could you please provide more info on the need of this event or even better, describe the scenario that you are trying to achieve?

**Comment by System64** [comment]
@bampakoa: change call back has many uses, like in my cases, I would like to perform some activity if there is any changes in the editor, like disable some other fields or change some texts, or show count of characters.

**Comment by bampakoa** [comment]
You could detect any changes that happen on the editor by watching **ngModel** attribute in your code:

**Comment by System64** [comment]
Yes watching ngModel can be done but don't you think it will increase the number of watcher for angular.

**Comment by bampakoa** [comment]
I don't think you have to worry so much about that, as the latest version of Angular >= 1.5 has many performance improvements. Nevertheless, it always depend on your code (if you are using already many watches etc....). Also, it would be better to watch the **model** rather than polluting the **rootScope** with an event.

**Comment by System64** [comment]
Yeah, that is the whole reason I am avoiding using it with watch, there are already a lot of things being watched.

No, I don't have to put it in rootScope, I have bind it with JQuery 's "DOMSubtreeModified" event.
